public/src/components/Home/ArchivedSubredditsComponent.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import ArchivedSubredditItem from './ArchivedSubredditItem';
+
+const ScrollArea = require('react-scrollbar').default;
```

Comment:
`import`?

Reply:
got it. fixed in https://github.com/YuriyBC/Reddit-archive/pull/1/commits/9b0510887ebaa3ee9c019b5ce302eebe3b8dd939